celext/celfuncs.go

```diff
@@ -58,6 +66,46 @@ func k8sIsHealthy() cel.EnvOption {
 	)
 }
 
+func jsonArrayDump() cel.EnvOption {
+	return cel.Function("data.JSONArrayDump",
```

Comment:
can we remove the prefix and use camelCase e.g. toJson ? with type overloading ?

Don't we already have this method from gomplate e.g. https://github.com/flanksource/gomplate/blob/main/funcs/data_gen.go#L29 ?

Reply:
> Don't we already have this method from gomplate e.g. https://github.com/flanksource/gomplate/blob/main/funcs/data_gen.go#L29 ?

This does not work as we want it to, we want to return a string type and the function should accept a dyn type which then needs to be converted to map[string]any. Also, that function returns the wrong result. It converts the dyn type to string incorrectly

> can we remove the prefix and use camelCase e.g. toJson ? with type overloading ?
cool, will try that

Reply:
Why do we need the dyn in the first place?

If those functions don't work, we should add tests for them and then exclude them so that we know its WIP

Reply:
> Why do we need the dyn in the first place?
Its not a hard requirement. `dyn` supports functions like `map, filter` etc. so as a user it is easy to work with that. We can accept a any type as well here as well, it wont break anything

>If those functions don't work, we should add tests for them and then exclude them so that we know its WIP
Have created an issue to track that: https://github.com/flanksource/gomplate/issues/20